src/plugins/calculationTry1/countCellsData.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export default function countCellsData(prop: JSX.Element) {
+  function handleKeyPress(e: React.KeyboardEvent<HTMLElement>) {
+    let countString = (e.target as Element).innerHTML;
+    if (e.key === "Enter" && countString[0] === "=" && (e.target as Element).classList.contains("data-input")) {
+      e.preventDefault();
+
+      if (!countString.slice(1).match(/[^0-9.+-/*()]/g)) {
+        let result: number;
+        try {
+          result = eval(countString.slice(1));
+        } catch (err) {
+          alert("Проверьте формулу, она некорректна");
+        }
+        if (!(isNaN(result) || result === undefined)) {
+          (e.target as Element).innerHTML = result.toString();
+        }
+      } else {
+        alert(
```

Comment:
use `messages.*()` instead of `alert()`

Reply:
We don't have styles for antd components yet. Should I solve it?

Reply:
yep